Library/Homebrew/utils/gems.rb

```diff
@@ -12,6 +12,8 @@ module Homebrew
   # After updating this, run `brew vendor-gems --update=--bundler`.
   HOMEBREW_BUNDLER_VERSION = "2.3.26"
 
+  VALID_GEM_GROUPS = ["sorbet", "prof"].freeze
```

Comment:
Could this be read from the `Gemfile` instead of being hardcoded?

Reply:
Will have a look and see if there's a public API for that.

Reply:
We already use some `Gem` methods so should be doable.